### PR TITLE
Azerbaijani translation has been added

### PR DESCRIPTION
**Pull request by nalekberov**

### CODE REVIEW

**Comment by stof** [comment]
We don't accept contributions on the i18n.php file itself (as explained in our contributing guidelines), because this file is generated automatically based on the translations available in cucumber's gherkin project. Translation contributions should go there instead: https://github.com/cucumber/gherkin/blob/master/gherkin-languages.json

**Comment by stof** [comment]
Please send a PR to the cucumber/gherkin repo instead

File: i18n.php [hunk 1]
```diff
@@ -59,6 +59,21 @@
     'then' => 'اذاً|ثم',
     'when' => 'عندما|متى',
   ),
+  'az' => 
+  array (
+    'and' => 'Və/Həm',
```
Comment (stof) [inline i18n.php]: it is expected to have a ``/`` here ? shouldn't it be 2 alternatives (and so a ``|``) ?

But this allowed me to catch a bug in gherkin btw.

**Comment by stof** [comment]
I'm closing this PR as it cannot be accepted here

File: i18n.php (hunk 1 above)
Comment (nalekberov) [inline i18n.php]: My bad. It is not allowed I know. Didn't check. By the way I sent PR to cucumber/gherkin and they merged it.

**Pull request closed without merging**